packages/date-time/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -1,23 +1,22 @@
 import * as React from 'react';
 import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
-import { DatePicker, DayOfWeek, IDatePickerStrings } from 'office-ui-fabric-react/lib/DatePicker';
+import { DatePicker, DayOfWeek, IDatePickerStrings } from '../../../DatePicker';
```

Comment:
same note about not doing relative imports as much as possible because it isn't what people can copy & paste in our example pages.